build.gradle

```diff
@@ -24,7 +24,7 @@ allprojects {
 
     }
 
-    project.version = "1.0.0-beta10-SNAPSHOT"
+    project.version = "1.0.0-beta10"
```

Comment:
bu development should always have X.X.X-SNAPSHOT. so you going to update it again?

Reply:
yes I was only planning to merge back but I guess easier to update here itself for SNAPSHOT. Done.